lib/tasks/limber.rake

```diff
@@ -122,6 +122,33 @@ namespace :limber do
         asset_shape: AssetShape.find_by(name: 'Standard')
       )
     end
+
+    unless Purpose.where(name: 'TR Stock 96').exists?
+      TubeRack::Purpose.create!(
+        name: 'TR Stock 96',
+        target_type: 'TubeRack',
+        stock_plate: true,
+        default_state: 'pending',
+        barcode_printer_type: BarcodePrinterType.find_by(name: '96 Well Plate'),
+        cherrypickable_target: false,
+        size: 96
+      )
+    end
+
+    unless Purpose.where(name: 'TR Stock 48').exists?
+      TubeRack::Purpose.create!(
+        name: 'TR Stock 48',
+        target_type: 'TubeRack',
+        stock_plate: true,
+        default_state: 'pending',
+        barcode_printer_type: BarcodePrinterType.find_by(name: '96 Well Plate'),
```

Comment:
Are these supposed to be the same? Maybe a comment as to why they are?

Reply:
What do you mean by these? The block above and this block? They have different names and sizes - 96 vs 48